coops-ui/src/components/AnnualReport/Directors.vue

```diff
@@ -981,6 +1019,11 @@ export default class Directors extends Mixins(DateMixin, ExternalMixin, EntityFi
     // close standalone cessation date picker and reset date
     this.cessationDateTemp = null
     this.activeIndexCustomCease = null
+    console.log(this.directors)
+    if (this.directors.filter(x => x.cessationDate == null).length < 2) {
+      this.lastValidDirector = director.id
+    }
+    // Determine if there are enough directors to remain in compliance
```

Comment:
Obsolete comment or incomplete code?

Reply:
not even used I should have deleted these 4 lines entirely